src/PharoThings-Devices-HD44780.package/PotLCD.class/instance/busMode..st

```diff
@@ -0,0 +1,3 @@
+accessing
+busMode: aSymbol
+	busMode := aSymbol
```

Comment:
What is it?

Reply:
This is to set if you want use 4 Gpios to send 4 data bits (split the 8 bits) or if you want to use 8 Gpios to send the 8 bits at the same time. If is set 4 bits, the index in "write: charMode:" is reset to use the 4 wires data again.